src/game/client/components/mapimages.cpp

```diff
@@ -196,3 +147,89 @@ int CMapImages::GetOverlayCenter()
 {
 	return m_OverlayCenterTexture;
 }
+
+void CMapImages::TextureSize(int Size)
+{
+	if (m_TextureSize == Size)
+		return;
+
+	m_TextureSize = Size;
+
+	if (Graphics()) // check if component was initialized
```

Comment:
```suggestion
	if(Graphics()) // check if component was initialized
```
Style

I think you should/could also check `m_OverlayBottomTexture == -1`